packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -1,19 +1,36 @@
 import React from 'react';
+import Joyride, { LIFECYCLE } from 'react-joyride';
+import { observer } from 'mobx-react-lite';
+import { useHistory } from 'react-router-dom';
 import CFDsListing from 'Components/cfds-listing';
 import ModalManager from 'Components/modals/modal-manager';
 import MainTitleBar from 'Components/main-title-bar';
 import OptionsAndMultipliersListing from 'Components/options-multipliers-listing';
-import './traders-hub.scss';
-import { DesktopWrapper, MobileWrapper, ButtonToggle, Div100vhContainer } from '@deriv/components';
+import ButtonToggleLoader from 'Components/pre-loader/button-toggle-loader';
```

Comment:
since we are wrapping all the containers in the index file, better to move all the logic of tour guide in a separate file and import that component here.
```suggestion
import TourGuide from 'Components/tour-guide/tour-gide';
```